tool/run/run

```diff
@@ -956,10 +956,20 @@ proc build_initrd { modules } {
     cd [run_dir]
     puts "generating initrd"
     exec cp genode/initramfs init
+    exec cp [genode_dir]/tool/boot/hwio.ko hwio.ko
+    exec cp [genode_dir]/tool/boot/platform_info.ko platform_info.ko
     exec mkdir tmp
-    set files "init\ntmp\ngenode\n"
+    exec mkdir dev
+    set files "init\nhwio.ko\nplatform_info.ko\ntmp\ndev\ngenode\n"
     append files [exec find genode -type f,l -printf "genode/%f\n"]
     exec -ignorestderr echo $files | cpio -o -L -H newc > initrd
+    
+    #workaround because cpio fails to compress broken links sometimes
+    exec touch dev/platform_info
+    cd genode
+    exec ln -s ../dev/platform_info platform_info
+    cd ..
+    exec -ignorestderr echo "genode/platform_info" | cpio -o -A -H newc -O initrd
```

Comment:
Use `installed_command` for cpio

Reply:
Was this already available in 17.08? Grepping through the repo doesn't find any occurence of this string.

Reply:
It was previously called `check_installed`. As this name is a bit misleading, we have recently renamed to `installed_command`. The corresponding issue is https://github.com/genodelabs/genode/issues/2853

Reply:
For now I would use `check_installed` and update it once this branch is rebased to the current version of Genode. I want to keep a working state in this branch.